src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -1,10 +1,13 @@
 # Compiling with Native AOT
 
-This document explains how to compile and publish your project using Native AOT toolchain. First, please _ensure that [pre-requisites](prerequisites.md) are installed_. If you are starting a new project, you may find the [HelloWorld sample](https://github.com/dotnet/samples/tree/main/core/nativeaot/HelloWorld/README.md) directions useful.
+Please consult [documentation](https://docs.microsoft.com/dotnet/core/deploying/native-aot) for instructions how to compile and publish application.
```

Comment:
Could you please also delete https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/docs/prerequisites.md ? It is no longer referenced now that this is deleted.